src/helpers/tests/executionResultHelper.spec.ts

```diff
@@ -230,3 +232,61 @@ describe("getHeaderItemName", () => {
         expect(getHeaderItemName(headerItems[1][0][0])).toBe("");
     });
 });
+
+describe("getHeadersInDimension", () => {
+    const executionResponse = require("../../../stories/test_data/geo_chart/geo_chart_with_location_size_color_segment_tooltip_response.json")
```

Comment:
please put this json file into fixture, and re-use :)

Reply:
This file only available in `/gooddata-react-components/mock-schema.js` for mock server and here.
So I think it would be better to import it directly than in `fixtures`